src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -136,26 +136,27 @@ async function writePackageJson(
     throw e;
   }
 
-  // Removes scripts from package.json after build
-  if (packageJson.scripts) {
-    if (pkg.keepLifecycleScripts !== true) {
-      log.info(`Removing scripts section in package.json as it's considered a potential security vulnerability.`);
+  // Process package.json scripts.
+  if (pkg.keepLifecycleScripts) {
+    log.warn(
```

Comment:
We shouldn’t show the warning unless scripts exists.

Reply:
You are right. Wrapped this in a condition.